tests/testthat/test_paramtest_survival_surv_parametric.R

```diff
@@ -35,17 +35,18 @@ test_that("paramtest surv.parametric train", {
 
 test_that("paramtest surv.parametric predict", {
   learner = lrn("surv.parametric")
-  fun = .predict_survreg
   exclude = c(
     # handled via mlr3
     "type",
     "object",
     "task",
     "feature_names",
     "tobit",
-    "learner"
+    "learner",
+    "discrete"
   )
 
-  paramtest = run_paramtest(learner, fun, exclude, tag = "predict")
+  paramtest = run_paramtest(learner, fun = .predict_survreg_continuous, exclude, tag = "predict")
```

Comment:
fun should be list containing both functions

Reply:
refactored